commands/builds/create.js

```diff
@@ -9,11 +9,18 @@ let request = require('request')
 let exec = require('child_process').execSync
 let nodeTar = require('../../lib/node_tar')
 
-function compressSource (tar, cwd, tempFile, cb) {
+function compressSource (context, tar, cwd, tempFile, cb) {
   let tarVersion = exec(tar + ' --version').toString()
 
   if (tarVersion.match(/GNU tar/)) {
-    exec(tar + ' cz -C ' + cwd + ' --exclude .git --exclude .gitmodules . --exclude-vcs-ignores > ' + tempFile)
+    let excludeVcsIgnore = context.flags['exclude-vcs-ignore'] || 'true'
+    let command = tar + ' cz -C ' + cwd + ' --exclude .git --exclude .gitmodules .'
+
+    if (excludeVcsIgnore === 'true') {
```

Comment:
I would have prefered to keep it the same command name. But that should work. Done in 6754b15daf3f3e97da9d207c6ed1d3682df9ec08.